pkg/models/trb_advice_letter_recommendation.go

```diff
@@ -12,5 +14,6 @@ type TRBAdviceLetterRecommendation struct {
 	Title            string         `json:"title" db:"title"`
 	Recommendation   HTML           `json:"recommendation" db:"recommendation"`
 	Links            pq.StringArray `json:"links" db:"links"`
-	PositionInLetter int            `json:"positionInLetter" db:"position_in_letter"` // 0-based indexing
+	PositionInLetter *int           `json:"positionInLetter" db:"position_in_letter"` // 0-based indexing
```

Comment:
Instead of an `*int`, would it be better to use a `null.Int`? We usually use that package when possible.

Reply:
(Oddly, that package also has a [null.Time](https://pkg.go.dev/github.com/guregu/null@v4.0.0+incompatible#Time) type, but we use `*time.Time` almost everywhere instead)

Reply:
37a62e01a